coc/http.py

```diff
@@ -34,7 +34,7 @@
 from urllib.parse import urlencode
 from itertools import cycle
 from datetime import datetime
-from .errors import HTTPException, Maitenance, NotFound, InvalidArgument, InvalidToken, Forbidden
+from .errors import HTTPException, Maitenance, NotFound, InvalidArgument, InvalidKey, Forbidden
```

Comment:
I'm happy to change this in .errors, .__init__, and docs (docs/api.rst) (renaming InvalidToken --> InvalidKey)

Reply:
No need for it I just got rid of it